File: Lec-1/Assignment/Cpp/Sum_and_Avg.cpp
#include <iostream>
using namespace std;

int main()
{
    int size;
    cout<<"Enter the size of the array: ";
    cin>>size;

    int arr[size];
    cout<<"Insert the elements in the array: ";
    for(int i=0;i<size;i++)
    {
        cin>>arr[i];
    }
    int sum=0;
    for(int i=0;i<size;i++)
    {
        sum+=arr[i];
    }
    cout<<"The sum of the elements of the array: "<<sum<<endl;
    cout<<"The avg of the elements of the array: "<<(sum/size);
}

File: Lec-1/Assignment/Java/Sum_and_Avg.java
package Assignment.Java;

import java.util.Scanner;

public class Sum_and_Avg
{
    public static void main(String[] args)
    {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter the size of the array: ");
        int size=sc.nextInt();
        int[] array=new int[size];
        System.out.print("Insert the Elements in the array: ");
        for(int i=0;i<size;i++)
        {
            array[i]=sc.nextInt();
        }

        int sum=0;
        System.out.print("Elements in the array: ");
        for(int i=0;i<size;i++)
        {
            System.out.print(array[i]+" ");
        }

        for(int i=0;i<size;i++)
        {
            sum=sum+array[i];
        }
        System.out.println("\nSum of the array: "+sum);

        System.out.println("Average of the Array: "+(sum/size));
        sc.close();
    }

}
